src/PeselValidator/PeselValidator.cs

```diff
@@ -26,13 +28,16 @@ public static bool IsValid(string pesel)
 
         private static bool HasValidDay(string pesel)
         {
+            var b = 0;
+
             var day = int.Parse(pesel.Substring(4, 2));
 
             return 1 <= day && day <= 31;
         }
 
         private static bool IsEmptyOrNotProperLength(string pesel)
         {
+            var c = 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'c' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)